packages/RNTester/metro.config.js

```diff
@@ -6,7 +6,7 @@
  */
 
 const path = require('path');
-const reactNativePath = path.resolve('../..');
+const reactNativePath = path.resolve(__dirname, 'node_modules', 'react-native');
```

Comment:
This change should also not be committed. Unlinking React Native is not the recommended method for using RNTester.

Reply:
yeah, messed up while merging changes :sweat_smile: